common/src/main/java/com/wynntils/handlers/chat/ChatHandler.java

```diff
@@ -282,6 +331,16 @@ private void processNewLines(LinkedList<StyledText> newLines) {
             // This is an NPC dialogue screen.
             // First remove the "Press SHIFT/Select an option to continue" trailer.
             newLines.removeFirst();
+
+            // If this happens, the "Press SHIFT/Select an option to continue" got appended to the last dialogue
+            // NOTE: Currently, we do nothing in this case, as it seems to work without any issues
```

Comment:
Did you encounter any situation where a confirmationless dialogue was turned into a confirmation-requiring one?

Reply:
Interesting. Do you remember any specific situation (a quest or some other instance)? I never noticed this, but maybe it is a new thing in Wynncraft as well. Or I just did not look around in enough places...

Reply:
[Kooha-2024-07-28-10-43-39.webm](https://github.com/user-attachments/assets/186c2dbe-dfd9-4c37-9aba-8dc78d0401ef)

I only have a video of these special confirmationless dialogues, that are basically time or distance based. `Hero of Gavel` has quite a few dialogues that are regular NPC dialogues, they start out as confirmationless, and they turn out to be SHIFT to continue (with a time based auto-continue in most cases).

Reply:
I need to go and check that in play myself. I wonder if I have a character with enough levels that have not done Hero of Gavel...

Reply:
Iirc Hollow Serinity has them tok and it's way less levels (but Hero of Gavel is much better for testing). Hollow Serentiy however has NPCs with special selection dialogues that are still broken :)

Reply:
So basically what you are saying is that if/when I get the time to work with the dialogue system again, I should replay Hollow Serenity and fix so it works, and then I also get the chance to check out these transmogrifying dialogues. :) I'll bear that in mind.